mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -356,7 +353,7 @@ function resultsToTables(results, target, uri) {
     tableid = "#" + tableid;
     $(target).html(html);
     $(target).attr('data', uri);
-    //retreive albumcovers
+    //retrieve albumcovers
     for (i = 0; i < coversList.length; i++) {
         getCover(coversList[i][0], target + '-cover-' + coversList[i][1], 'small');
```

Comment:
The whole ``resultsToTables`` function is a little hard to follow and probably due for some refactoring (I'm not sure where to start though).

Until I have a better understanding of what it does and how it works: I'm not sure that a ``getImages`` will be useful anywhere else. But I did notice that there are a lot of calls go ``getCover`` for just about every navigation action in the client, so doing it once and caching the results for retrieval later on would most likely make more sense.